Spelling and Grammar Updates (#23)
SheCodesAustralia/workshopcontent: content/python/space_turtle_chomp/part_11__60_second_countdown.md

else:
    mypen.setposition(0, 0)
    mypen.color("yellow")
    mypen.write("Game Over: You LOSE", False, align="center", font=("Arial", 28, "normal"))
```

The if statement compares your score \(player\) against the opponent score
 \(comp\) and if your score is higher it prints the You Win message and if it
 is lower it prints the You Lose message.

> If you run your code now, the game will quit before you can actually read the messge so lets add a delay so we can see the message.

Reviewer: ```suggestion
If you run your code now, the game will quit before you can actually read the message. Let’s add a delay so we can see the message.
```

```python
delay = input("Press Enter to finish.")
```

That is the end of today’s tutorial, if you got through all of it in 1 day well
 done that is an awesome effort but it doesn’t matter if you didn’t as the
 tutorial is online and you can still access it from home. Don't forget to join

>  our [She Codes online community] (https://join.slack.com/t/shecodesaus/shared_invite/zt-9jktxnlx-Rur3NGFSBFJ7LRwq7AV~ig) to keep in touch with your mentors and ask for help.

Reviewer: ```suggestion
 our [She Codes online community](https://join.slack.com/t/shecodesaus/shared_invite/zt-9jktxnlx-Rur3NGFSBFJ7LRwq7AV~ig) to keep in touch with your mentors and ask for help.
```

* Setting a play again option at the end
* Setting up a easy, medium and hard option for the game

There are lots of tutorials and help out on the internet.

>  It is also a great idea to try the other game options from Christian Thompson
>  at his site [http://christianthompson.com](http://christianthompson.com)

Reviewer: ```suggestion
 It is also a great idea to try the other game options from Christian Thompson
 at his [site](http://christianthompson.com).
```
